src/firebase.js

```diff
@@ -0,0 +1,29 @@
+// import firebase from 'firebase/app';
```

Comment:
why is this file commented?

Reply:
sorry - that was me. You can just delete this for now. IT was added to deal with a cyclical import.

Reply:
the whole firebase.js is commented. what should I do?

Reply:
You can remove the entire file.

Reply:
on second thought, i will do a cleanup while writing some tests. don't worry about this.